tests/NCronJob.Tests/ExtensionsTests.cs

```diff
@@ -1,77 +1,77 @@
-using Microsoft.Extensions.DependencyInjection;
-using Shouldly;
+//using Microsoft.Extensions.DependencyInjection;
```

Comment:
This whole file is *temporarily* commented out as I haven't fixed the tests. Yet.

Reply:
Fixed